ci-operator/config/openshift/oc/openshift-oc-master.yaml

```diff
@@ -100,6 +100,11 @@ resources:
     requests:
       cpu: 100m
       memory: 100Mi
+  cli-artifacts:
```

Comment:
Wait.. your requests before were 6Gi?  Something weird is going on then, you should not have been oomkilled...